service/frontend/adminHandler.go

```diff
@@ -1038,7 +1038,7 @@ func (adh *AdminHandler) DescribeCluster(
 		ClusterName:              metadata.ClusterName,
 		HistoryShardCount:        metadata.HistoryShardCount,
 		PersistenceStore:         adh.persistenceExecutionManager.GetName(),
-		VisibilityStore:          adh.visibilityMgr.GetName(),
+		VisibilityStore:          strings.Join(adh.visibilityMgr.GetStoreNames(), ","),
```

Comment:
I see this is used in a couple of places. Should we have a function to retrieve the store names as string? Worried about fragmentation, and eventually someone calling `Join` with another separator by accident.